examples/common/utils.py

```diff
@@ -17,6 +17,28 @@
 # Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 # ==============================================================================
+# MIT License
```

Comment:
pls delete this license text for now and revert it to earlier stage, as the source links are already mentioned. pls do it for all the 3 files which has this license text.

Reply:
removed the other license text